scope/init.go

```diff
@@ -106,10 +106,18 @@ func Init(my *Extent, create bool, version string) (*Extent, error) {
 		return nil, err
 	}
 
-	if _, err = ReadVersion(my, sv); create && err != nil {
-		// only set default version if a version is not present
-		if err = setDefaultVersion(my, sv, version); err != nil {
-			return nil, err
+	if create {
+		log.Printf("# -> Force: %t", force)
+		if _, err = ReadVersion(my, sv); err != nil {
```

Comment:
Should this `ReadVersion` get execute in the else branch? I.e. do `if force {...` first.   From the current logic, it seems that it won't set the version even if `force` flag is true and there is an existing version can be read.

Reply:
I updated code to utilize a logical OR, we need to ```ReadVersion``` only once to check if there is a version already present.

Reply:
ok, now it will do the required logic when logic is true. The old code won't do it if readversion is read ok even if the force flag is true since it was in the `else if`  LGTM